src/styles/index.scss

```diff
@@ -2,2 +1,19 @@
+@import 'variables';
+@import 'header';
+@import 'nav';
+@import 'catalog';
+@import 'card';
+@import 'star';
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  line-height: 14px;
```

Comment:
The font-size and line-height values are set in pixels. It's generally recommended to use relative units like 'rem' or 'em' for better scalability and accessibility.